app/src/main/java/co/touchlab/kampstarter/android/BreedViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package co.touchlab.kampstarter.android
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.MutableLiveData
+import co.touchlab.kampstarter.db.Breed
+import co.touchlab.kampstarter.models.BreedModel
+import kotlinx.coroutines.InternalCoroutinesApi
+import androidx.lifecycle.viewModelScope
+import co.touchlab.kampstarter.models.ItemDataSummary
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+@InternalCoroutinesApi
```

Comment:
Fairly certain these annotations aren't necessary. If they are, we should be doing `@OptIn(InternalCoroutinesApi::class)` at the source instead of leaking `@InternalCoroutinesApi` up the whole call-chain.

Reply:
Removed